projects/quml-library/src/lib/services/viewer-service/viewer-service.ts

```diff
@@ -315,13 +315,13 @@ export class ViewerService {
     videoElements.forEach((element: HTMLVideoElement) => element.pause());
   }
 
-  serverValidationCheck(obj: any) {
-    if(typeof obj == 'string') {
-      this.questionSetEvaluable = JSON.parse(obj);
-      this.questionSetEvaluable = this.questionSetEvaluable?.mode?.toLowerCase() == 'server'
+  serverValidationCheck(mode: any) {
```

Comment:
@pavankumar0408 Instead of returning the `true` / `false` from this method can you just re-assign the value of `questionSetEvaluable` and use the value in all other places in components. So this way we can reduce the call of same method in multiple places.